guides/common/modules/proc_adding-a-host-to-a-host-collection-by-using-cli.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-a-host-to-a-host-collection-by-using-cli"]
+= Adding a host to a host collection by using Hammer CLI
+
+[role="_abstract"]
+You can add a host to a host collection by using Hammer CLI.
+
+.Prerequisites
+* A host must be registered to {ProjectName} to add it to a Host Collection.
+For more information about registering hosts, see xref:Registering_Hosts_by_Using_Global_Registration_{context}[].
++
```

Comment:
I don't know if this is due to DITA, but the following line does not really below to prerequisites. -> 

```suggestion

```

If the `+` is due to DITA, I suggestion to move it below line 8 instead.

Reply:
Not a DITA thing, just my oversight ;) Thanks, moved.